tests/workflows/components/workflows.libsonnet

```diff
@@ -46,7 +38,7 @@
       local srcRootDir = testDir + "/src";
       // The directory containing the kubeflow/manifests repo
       local srcDir = srcRootDir + "/kubeflow/manifests";
-      local testWorkerImage = "gcr.io/kubeflow-ci/test-worker-py3:e9afed1-dirty";
+      local testWorkerImage = "527798164940.dkr.ecr.us-west-2.amazonaws.com/aws-kubeflow-ci/test-worker:latest";
```

Comment:
Is there a stable tag to use? Let's try not to use latest.

Reply:
Emm let me try to cut a stable tag and replace here

Reply:
Use `527798164940.dkr.ecr.us-west-2.amazonaws.com/aws-kubeflow-ci/test-worker:v1.1-branch` instead